File: URI/1048.c
#include<stdio.h>
#include<stdlib.h>

float N,fim,reajuste;
int p;

int main(){
	scanf("%f",&N);
	if(N>=0.0 && N<=400.00){
		reajuste=N*0.15;
		fim=N+reajuste;
		p=15;
	}else if(N>=400.01 && N<=800.00){
		reajuste=N*0.12;
		fim=N+reajuste;
		p=12;
	}else if(N>=800.01 && N<=1200.00){
		reajuste=N*0.10;
		fim=N+reajuste;
		p=10;
	}else if(N>=1200.01 && N<=2000.00){
		reajuste=N*0.07;
		fim=N+reajuste;
		p=7;
	}else if(N>2000.00){
		reajuste=N*0.04;
		fim=N+reajuste;
		p=4;
	}
	printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %d %%\n",fim,reajuste,p);
	return 0;
}

File: URI/1048.py
salario = float(input())
novo_salario = 0
p = 0

if salario >= 0.0 and salario <= 400.00:
	novo_salario = salario + (salario * 0.15)
	p = 15
elif salario >= 400.01 and salario <= 800.00:
	novo_salario = salario + (salario * 0.12)
	p = 12
elif salario >= 800.01 and salario <= 1200.00:
	novo_salario = salario + (salario * 0.1)
	p = 10
elif salario >= 1200.01 and salario <= 2000.00:
	novo_salario = salario + (salario * 0.07)
	p = 7
elif salario > 2000.00:
	novo_salario = salario + (salario * 0.04)
	p = 4

print("Novo salario: {:.2f}".format(novo_salario))
print("Reajuste ganho: {:.2f}".format(novo_salario - salario))
print("Em percentual: {} %".format(p))